tests/solvers/test_radau5.py

```diff
@@ -26,8 +25,9 @@
 from assimulo.exception import *
 from assimulo.lib.radau_core import Radau_Exception, Radau_Common
 import scipy.sparse as sp
-import numpy as np
+import numpy as N
```

Comment:
Nothing you have to change but in general I think its better to import a module with at least two letters, because if you have to search for _N._ now you will likely find way more entries than _np._.

Reply:
While I also prefer np over N for that exact reason, I tried to keep it consistent with the rest of Assimulo, where it is predominantly import numpy as N.